sdk/openai/Azure.AI.OpenAI/src/Azure.AI.OpenAI.csproj

```diff
@@ -19,7 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
```

Comment:
Curious - can this line be removed now?

Reply:
It's sadly not that simple:
- We have a direct dependency on Azure.Core.TokenCredential, Azure.Core.AzureKeyCredential, etc...
- As part of the Azure repo, a link to Shared\Core\*.cs is automatically added to the project which has classes that all directly depend on Azure.Core